pkg/agent/datamodel/defaults-custom-cloud-profile.go

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
we don't need this

Reply:
PR to remove set custom cloud/azure stack defaults is here: https://github.com/Azure/AgentBaker/pull/195